documentation/src/docs/asciidoc/user-guide/appendix.adoc

```diff
@@ -39,6 +39,8 @@ artifacts are deployed to Sonatype's {snapshot-repo}[snapshots repository] under
     directory. See <<running-tests-console-launcher>> for details.
   `junit-platform-engine`::
     Public API for test engines. See <<launcher-api-engines-custom>> for details.
+  `junit-platform-jfr`::
+    Provides the JUnit Platform Flight Recording Listener. See <<running-tests-flight-recorder>> for details.
```

Comment:
How about something like "Provides a Java Flight Recorder `TestExecutionListener` for the JUnit Platform" or "Provides a `TestExecutionListener` for Java Flight Recorder events on the JUnit Platform"?

Reply:
I take the second sentence.